taskt/Core/Automation/Commands/IE Browser Commands/IEFindBrowserCommand.cs

```diff
@@ -83,27 +96,19 @@ public override List<Control> Render(frmCommandEditor editor)
             var shellWindows = new ShellWindows();
             foreach (IWebBrowser2 shellWindow in shellWindows)
             {
-                if (shellWindow.Document is MSHTML.HTMLDocument)
+                if (shellWindow.Document is HTMLDocument)
                     IEBrowerNameDropdown.Items.Add(shellWindow.Document.Title);
             }
             RenderedControls.Add(CommandControls.CreateDefaultLabelFor("v_IEBrowserName", this));
             RenderedControls.AddRange(CommandControls.CreateUIHelpersFor("v_IEBrowserName", this, new Control[] { IEBrowerNameDropdown }, editor));
-            //IEBrowerNameDropdown.SelectionChangeCommitted += seleniumAction_SelectionChangeCommitted;
             RenderedControls.Add(IEBrowerNameDropdown);
 
-            //ElementParameterControls = new List<Control>();
-            //ElementParameterControls.Add(CommandControls.CreateDefaultLabelFor("v_WebActionParameterTable", this));
-            //ElementParameterControls.AddRange(CommandControls.CreateUIHelpersFor("v_WebActionParameterTable", this, new Control[] { ElementsGridViewHelper }, editor));
-            //ElementParameterControls.Add(ElementsGridViewHelper);
-
-            //RenderedControls.AddRange(ElementParameterControls);
-
             return RenderedControls;
         }
 
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [Browser Name: '" + v_IEBrowserName + "', Instance Name: '" + v_InstanceName + "']";
+            return base.GetDisplayValue() + $" [Attach to Browser '{v_InstanceName}' Having Title '{v_IEBrowserName}']";
```

Comment:
Please change to [Having Title '{v_IEBrowserName}' - Instance Name '{v_InstanceName}']

Reply:
done